Update upgrade-distribution-lists.md (#815)
MicrosoftDocs/OfficeDocs-O365ITPro: Office365-Admin/manage/upgrade-distribution-lists.md

> [!NOTE]
> If you're getting the groups digest emails you may notice at the bottom that it will sometimes offer to let you upgrade any eligible distribution lists that you're the owner of. See [Have a group conversation in Outlook](https://support.office.com/article/a0482e24-a769-4e39-a5ba-a7c56e828b22.aspx) for more information about digest emails.

> [!NOTE]

> > The Distribution group must have an owner and the owner must have mailbox in order to upgrade to O365 groups.

Reviewer: ```suggestion
> The distribution group must have an owner and the owner must have a mailbox in order to upgrade to O365 Groups.
```

> > The Distribution group must have an owner and the owner must have mailbox in order to upgrade to O365 groups.

Reviewer 2: ```suggestion
> To upgrade to Office 365 groups, a distribution group must have an owner with a mailbox. 
```

> 

Reviewer 2: ```suggestion
```
